docs/advanced/witness.md

```diff
@@ -0,0 +1,82 @@
+
+```
+NAME↑               STATUS   ROLE                         VERSION               PODS     CPU      MEM    %CPU    %MEM    CPU/A    MEM/A AGE        
+harvester-node-0    Ready    control-plane,etcd,master    v1.27.10+rke2r1         70    1095    10143      10      63    10000    15976 4d13h      
+harvester-node-1    Ready    etcd                         v1.27.10+rke2r1          7     258     2258       2      14    10000    15976 4d13h      
+harvester-node-2    Ready    control-plane,etcd,master    v1.27.10+rke2r1         36     840     6905       8      43    10000    15976 4d13h      
+```
+
+You can see the `harvester-node-1` use less of resource to running as expection.
+
+## Workloads on the witness node
+
+The witness node only runs the must workload like etcd. Now the following list is the supported workloads on the witness node.
+- **harvester-node-manager**
+- **cloud-controller-manager**
+- **etcd**
+- **kube-proxy**
+- **rke2-canal**
+- **rke2-multus**
+
+## Upgrade the cluster with witness node
+
+When you upgrade the cluster with a witness node, you can upgrade the cluster as usual. But if you have a degraded volume, the upgrade might fail. You can check the following limitations for the volume robustness.
+
+## Limitations
+
+There were some limitations for the cluster with witness.
+
+- Volume robustness: The witness node does not store any data. It means we will miss one replica when the cluster has the witness node due to the Longhorn default replica number (3). The upgrade might failed by the degraded volume.
+
+  To resolve this, you should create a new StorageClass and decrease its replica number from 3 -> 2. Then, set this StorageClass as a default StorageClass.
+  ![new storageclass replica 2](/img/v1.3/advanced/new-storageclass-rep-2.png)
+  ![set to default](/img/v1.3/advanced/set-to-default-sc.png)
+
+  If you already create some volume with the default StorageClass, you can manually decrease the replica counts from 3 -> 2 on Longhorn UI -> **Volume Page** as a workaround. You can get the related info for how to redirect to Longhorn UI from [here](../troubleshooting/harvester/#access-embedded-rancher-and-longhorn-dashboards)
+  ![redirect-to-longhorn-volume-page](/img/v1.3/advanced/redirect-to-longhorn-vol-page.png)
+  ![update-replica-count-to-2](/img/v1.3/advanced/update-replica-2.png)
+
+  After, that the volume robustness will be healthy.
+
+
+## Known issues
+
+1. When you create a cluster with the witness node, the page of network config creation can not find any useable uplink with all nodes.
+![create network config with all nodes](/img/v1.3/advanced/create-policy-with-all-nodes.png)
+![no uplink](/img/v1.3/advanced/no-uplink.png)
+
+  You can choose nodes other than the witness node as a workaround. Then, you can successfully create the network config.
+  ![create network config with specific node](/img/v1.3/advanced/create-policy-with-specific-node.png)
```

Comment:
Please help add a note that the user needs to repeat this on all nodes that the network is supposed to span.

Reply:
updated!